Corona-Warn-App/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,1028 @@
+    <string name="information_title">"Информация о приложении"</string>
+    <!-- XTXT: Version information text -->
+    <string name="information_version">"Версия: %1s"</string>
+    <!-- XHED: Page title for about information page, also menu item / button text -->
+    <string name="information_about_title">"О приложении"</string>
+    <!-- XHED: Subtitle for about information page -->
+    <string name="information_about_headline">"Давайте победим коронавирус вместе"</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body_emphasized">"Институт Роберта Коха (RKI) — это центральный федеральный институт Германии в области здравоохранения. RKI выпустил приложение Corona-Warn-App для федерального правительства. Приложение стало цифровым дополнением к уже принятым санитарным мерам: социальной дистанции, гигиене и ношению защитных масок."</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body">"Каждый пользователь приложения помогает отслеживать и останавливать передачу инфекции. Приложение регистрирует контакты с другими людьми локально на вашем устройстве. Вы получаете уведомления о контактах с людьми, которые впоследствии получили положительный результат теста на COVID-19. Ваши персональные данные и конфиденциальность надежно защищены."</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_about_illustration_description">"Группа людей использует свои устройства в городе."</string>
+    <!-- XHED: Page title for privacy information page, also menu item / button text -->
+    <string name="information_privacy_title">"Конфиденциальность данных"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"На устройстве одной женщины установлено приложение Corona-Warn-App. Значок замка на щите указывает на шифрование данных."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_ru.html"</string>
+    <!-- XHED: Page title for terms of use information page, also menu item / button text -->
+    <string name="information_terms_title">"Условия использования"</string>
+    <!-- XHED: Page headline for terms of use information page -->
+    <string name="information_terms_headline">"Содержимое"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"Один человек держит устройство с длинным текстом на экране. Рядом с текстом отображается галочка, которую нужно установить для принятия условий использования."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_ru.html"</string>
```

Comment:
shouldn't that either
   * link to an existing file (https://github.com/corona-warn-app/cwa-app-android/tree/dev/Corona-Warn-App/src/main/assets)

or

   * terms_ru.html

be part of this pull request?